api/server/cluster.go

```diff
@@ -686,6 +698,10 @@ func clusterVersion(route, version string) string {
 	return "/" + version + "/" + route
 }
 
+func clusterSecretPath(route, version string) string {
+	return clusterVersion("cluster/secrets"+route, version)
```

Comment:
we can use clusterPath here
`return clusterPath("/secrets"+route, version)`

Also can we use the same path constants (secretPath - /secrets) that we used in client. May be define in server and then client uses that?